### PR TITLE
Add capital counts

### PR DESCRIPTION
**Pull request by jasonLaster**
Small UI fix with some small code tweaks

Screen Shot 2020-04-23 at 8 18 49 AM

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ require("dotenv").config({
 // project configuration.
 if (!process.env.AIRTABLE_BASE_ID) {
   throw new Error(
-    `AIRTABLE_BASE_ID property is missing from your env (.env.development file).
+    `AIRTABLE_BASE_ID property is missing from your env (.env.${process.env.NODE_ENV} file).
```
Comment (jasonLaster) [inline gatsby-config.js:11]: small code tweak

File: gatsby-node.js [hunk 2]
```diff
@@ -70,13 +70,13 @@ exports.createPages = async ({ graphql, actions }) => {
...
 
-  data.capitalTypes.nodes.forEach(({ id, data: { Slug: slug }}) => {
+  data.capitalTypes.nodes.forEach(({ id, data: { Slug: slug } }) => {
     createPage({
       path: `/capital/${slug}`,
       component: path.resolve(`./src/pages/capital.js`),
       context: {
         activeTypeId: id,
-        active: true
+        active: true,
```
Comment (jasonLaster) [inline gatsby-node.js:79]: prettier

**Comment by bloudermilk** [comment]
Screen Shot 2020-04-23 at 8 46 28 PM

Code looks good but I'm seeing a discrepancy here

**Comment by bloudermilk** [comment]
@jasonLaster would like to get this merged. Did you want to check out that bug or should I wrap this up?

**Comment by bloudermilk** [comment]
I think the issue here is that we're using the count from Airtable, which doesn't necessarily represent the same # of capital orgs that get published to the website. In order to implement this accurately we'll need to do something closer to `countCategoriesOrganizations` in `src/utils/gatsby.js`.

**Comment by bloudermilk** [comment]
Closing this as stale.

**Pull request closed without merging**